transform/mattermost-analytics/models/intermediate/web_app/int__performance_events.sql

```diff
@@ -0,0 +1,9 @@
+{{
+    dbt_utils.union_relations(
+        relations=[
+            ref('int_mm_telemetry_prod__performance_events'),
+            ref('int_mm_telemetry_rc__performance_events'),
+        ],
+        source_column_name=None
```

Comment:
Source would be nice to keep. The other examples I used already had a source field, so it would be duplicate information. That's the reason `source_column_name` is none for the event registry models.